### PR TITLE
Minor: remove the 5 from HTML

### PR DESCRIPTION
**Pull request by estelle**
Removed 5 from HTML.
Sometimes had to include a few words as a replacement.
Also, some HTML to MD conversions left a lot of carriage returns. I removed some.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/element/namespaceuri/index.md [hunk 1]
```diff
@@ -13,19 +13,15 @@ browser-compat: api.Element.namespaceURI
 ---
 {{APIRef("DOM")}}
 
-The **`Element.namespaceURI`** read-only property returns the
```
Comment (estelle) [inline files/en-us/web/api/element/namespaceuri/index.md:16]: this page is mostly just line returns

File: files/en-us/web/api/element/namespaceuri/index.md [hunk 2]
```diff
@@ -36,21 +32,13 @@ if (element.localName === "browser" &&
...
 
-In Firefox 3.5 and earlier, the namespace URI for HTML elements in HTML documents is
-`null`. In later versions, in compliance with HTML5, it is
-[`http://www.w3.org/1999/xhtml`](https://www.w3.org/1999/xhtml/)
-as in XHTML.
+The namespace URI for HTML elements in HTML documents is [`http://www.w3.org/1999/xhtml`](https://www.w3.org/1999/xhtml/) as in XHTML.
 
-You can create an element with the specified `namespaceURI` using the DOM
-Level 2 method [document.createElementNS](/en-US/docs/Web/API/Document/createElementNS).
+You can create an element with the specified `namespaceURI` using the DOM Level 2 method [document.createElementNS](/en-US/docs/Web/API/Document/createElementNS).
```
Comment (teoli2003) [inline files/en-us/web/api/element/namespaceuri/index.md:39]:
```suggestion
You can create an element with the specified `namespaceURI` using the [`document.createElementNS()`](/en-US/docs/Web/API/Document/createElementNS) method.
```

File: files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.md [hunk 3]
```diff
@@ -622,4 +622,4 @@ The following tables list basic audio support across desktop and mobile browsers
 - [Using HTML audio and video](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content)
 - [Creating a cross-browser video player](/en-US/docs/Web/Guide/Audio_and_video_delivery/cross_browser_video_player)
 - [jPlayer](https://jPlayer.org): An open source audio and video library for jQuery and Zepto.
-- [Media Event Inspector](https://jplayer.org/HTML5.Media.Event.Inspector/)
+- [jQuery Media Event Inspector](https://jplayer.org/HTML5.Media.Event.Inspector/)
```
Comment (teoli2003) [inline files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.md:625]: I remove this one there is already a link to jPlayer one line above. It is enough.
```suggestion
```

File: files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.md [hunk 4]
```diff
@@ -622,4 +622,4 @@ The following tables list basic audio support across desktop and mobile browsers
 - [Using HTML audio and video](/en-US/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content)
 - [Creating a cross-browser video player](/en-US/docs/Web/Guide/Audio_and_video_delivery/cross_browser_video_player)
 - [jPlayer](https://jPlayer.org): An open source audio and video library for jQuery and Zepto.
```
Comment (teoli2003) [inline files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.md:624]: NIT
```suggestion
- [jPlayer](https://jplayer.org): An open source audio and video library for jQuery and Zepto.
```

File: files/en-us/web/html/block-level_elements/index.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ There are a couple of key differences between block-level elements and inline el
...
+The distinction of block-level vs. inline elements was used in HTML specifications up to 4.01. In the HTML Living Standard, this binary distinction is replaced with a more complex set of [content categories](/en-US/docs/Web/Guide/HTML/Content_categories). While the "inline" category roughly corresponds to the category of [phrasing content](/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content), the "block-level" category doesn't directly correspond to any HTML content category, but _"block-level" and "inline" elements combined together_ correspond to the [flow content](/en-US/docs/Web/Guide/HTML/Content_categories#flow_content) in HTML. There are also additional categories, e.g. [interactive content](/en-US/docs/Web/Guide/HTML/Content_categories#interactive_content).
```
Comment (teoli2003) [inline files/en-us/web/html/block-level_elements/index.md:52]:
```suggestion
The distinction of block-level vs. inline elements was used in HTML specifications up to 4.01. Later, this binary distinction is replaced with a more complex set of [content categories](/en-US/docs/Web/Guide/HTML/Content_categories). While the "inline" category roughly corresponds to the category of [phrasing content](/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content), the "block-level" category doesn't directly correspond to any HTML content category, but _"block-level" and "inline" elements combined together_ correspond to the [flow content](/en-US/docs/Web/Guide/HTML/Content_categories#flow_content) in HTML. There are also additional categories, e.g. [interactive content](/en-US/docs/Web/Guide/HTML/Content_categories#interactive_content).
```

File: files/en-us/web/html/element/area/index.md [hunk 6]
```diff
@@ -144,7 +144,7 @@ This element's attributes include the [global attributes](/en-US/docs/Web/HTML/G
 
   - : Indicates that no hyperlink exists for the associated area.
 
-    > **Note:** Since HTML5, omitting the `href` attribute is sufficient.
+    > **Note:** `nohref` is not necessary, as omitting the `href` attribute is sufficient.
```
Comment (teoli2003) [inline files/en-us/web/html/element/area/index.md:147]:
```suggestion
    > **Note:** The `nohref` attribute is not necessary, as omitting the `href` attribute is sufficient.
```

File: files/en-us/web/html/element/form/index.md [hunk 7]
```diff
@@ -157,7 +157,7 @@ The following attributes control behavior during form submission.
   - : This Boolean attribute indicates that the form shouldn't be validated when submitted. If this attribute is not set (and therefore the form **_is_** validated), it can be overridden by a {{htmlattrxref("formnovalidate", "button")}} attribute on a {{HTMLElement("button")}}, [`<input type="submit">`](/en-US/docs/Web/HTML/Element/input/submit), or [`<input type="image">`](/en-US/docs/Web/HTML/Element/input/image) element belonging to the form.
 - {{htmlattrdef("target")}}
 
-  - : Indicates where to display the response after submitting the form. In HTML 4, this is the name/keyword for a frame. In HTML5, it is a name/keyword for a _browsing context_ (for example, tab, window, or iframe). The following keywords have special meanings:
+  - : Indicates where to display the response after submitting the form. In HTML 4, this was the name/keyword for a frame. In modern standards, it is a name/keyword for a _browsing context_ (for example, tab, window, or iframe). The following keywords have special meanings:
```
Comment (teoli2003) [inline files/en-us/web/html/element/form/index.md:160]:
```suggestion
  - : Indicates where to display the response after submitting the form. It is a name/keyword for a _browsing context_ (for example, tab, window, or iframe). The following keywords have special meanings:
```

File: files/en-us/web/html/element/object/index.md [hunk 8]
```diff
@@ -85,7 +85,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 - {{HTMLAttrDef("data")}}
   - : The address of the resource as a valid URL. At least one of **data** and **type** must be defined.
 - {{HTMLAttrDef("declare")}} {{deprecated_inline}}
-  - : The presence of this Boolean attribute makes this element a declaration only. The object must be instantiated by a subsequent `<object>` element. In HTML5, repeat the \<object> element completely each time that the resource is reused.
+  - : The presence of this Boolean attribute makes this element a declaration only. The object must be instantiated by a subsequent `<object>` element. In HTML, repeat the `<object>` element completely each time that the resource is reused.
```
Comment (teoli2003) [inline files/en-us/web/html/element/object/index.md:88]:
```suggestion
  - : The presence of this Boolean attribute makes this element a declaration only. The object must be instantiated by a subsequent `<object>` element. Repeat the `<object>` element completely each time the resource is reused.
```

**Review by teoli2003 — APPROVED** [review]
A few nits that I'll commit immediately.

**Pull request merged**